community/kernel/src/main/java/org/neo4j/kernel/IdType.java

```diff
@@ -19,14 +19,16 @@
  */
 package org.neo4j.kernel;
 
+import org.neo4j.unsafe.impl.internal.dragons.FeatureToggles;
+
 /**
  * @deprecated This will be moved to internal packages in the next major release.
  */
 @Deprecated
 public enum IdType
 {
     NODE( 35, false ),
-    RELATIONSHIP( 35, false ),
```

Comment:
Yes, I think this is a "normal" feature, not a special hidden thing. Instead of doing this inside the IdType enum the config thing could be done in the place using that value, right?